admin/customers.php

```diff
@@ -1183,7 +1183,7 @@ function check_form() {
                     <?php } else { ?>
                       <input type="image" src="<?php echo DIR_WS_IMAGES ?>icon_red_on.gif" title="<?php echo IMAGE_ICON_STATUS_OFF; ?>" />
                     <?php } ?>
-                    <?php echo zen_draw_hidden_field('current', $customer['customers_authorization']); ?>
+                    <?php echo zen_draw_hidden_field('current', strval($customer['customers_authorization'])); ?>
```

Comment:
why `strval()`?

Reply:
Oh. I see, you explained that in a separate commit.

Reply:
Actually, why not `(int)` it?

Reply:
I tried int, but for some mysterious reason that just doesn't work for 0.

Reply:
That does seem mysterious.

I try to avoid `strval` because it's less performant than a more explicit cast.